docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Create Cortex or Loki managed recording rule"
+description = "Create Cortex or Loki managed recording rule"
+keywords = ["grafana", "alerting", "guide", "rules", "recording rules", "create"]
+weight = 400
++++
+
+# Create a Cortex or Loki managed recording rule
+
+Grafana allows you manage recording rules for an external Cortex or Loki instance.
+
+Recording rules allow you to precompute frequently needed or computationally expensive expressions and save their result as a new set of time series. Querying the precomputed result will then often be much faster than executing the original expression every time it is needed. This is especially useful for dashboards, which need to query the same expression repeatedly every time they refresh.
+
+Recording and alerting rules exist in a rule group. Rules within a group are run sequentially at a regular interval, with the same evaluation time. The names of recording rules must be valid [metric names](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels). 
```

Comment:
```suggestion
Recording and alerting rules exist in a rule group and they are run sequentially at a regular interval with the same evaluation time. 
```

Reply:
you think the caveat about reacording rule names being valid metric names is not necesary?

Reply:
nvm, noticed it was just moved